Classes/Domain/Table.php

```diff
@@ -0,0 +1,151 @@
+<?php
```

Comment:
Never seen something like this.

Reply:
Actually I wanted to use enum not class but it turns out that PHP 7.x doesn't have yet access to it (https://stitcher.io/blog/php-enums).

The main idea here is to write string names of tables only once and use later only this variables instead of writing strings over and over again. I find it to be a bit less error prone.

Reply:
Ah, ok.

If "everything" (I doubt, that this is possible) is converted to Extbase, you do not enter the table name anywhere.

By-the-way: PHP 8 will not come to TYPO3 before version 12! Have a look at the roadmap and the PHP compatibility chart: https://typo3.org/cms/roadmap This would be Kitodo.Presentation 6.x (or more) ;-)